.env

```diff
@@ -1,6 +1,7 @@
 NODE_URL=ws://node:9944
-SMART_CONTRACT_ADDRESS=SMART_CONTRACT_ADDRESS
-APP_WALLET_JSON=APP_WALLET_JSON
-APP_WALLET_PASSPHRASE=APP_WALLET_PASSPHRASE
+SMART_CONTRACT_ADDRESS=5CgGJhgZdUHpsC6nHy3Y1iVpvtbW7EhBKvuAXE5WFsbmBVFk
+APP_WALLET_JSON={"address":"5EeJGvXetxrb2N33YdxZApd99i92uznpXaUdW8Nprfa2rqTS","encoded":"s6N+ZdLoXGdjCYmN3/jxfLwDTi1//CPeshwsSFrBXOkAgAAAAQAAAAgAAADz5ze2/IQsuBv2yhYP4gejynu9t8fIW+C5C8hNjj93NLCNSf2TZ/NLncVOyp+1BJe7yHW2MCy2AaFb14ZfbS7uyYY9qHyXbU1IOBiWZUX3qBCv2UhksEVrt67N3dH3b5maa9Uj/n9ywvPSqnTQREP5jt4drnWdHzp5OvTpGvxyghmKo9bUAmW8CL0j3PRVyldCkReFhms2iC1hhYbA","encoding":{"content":["pkcs8","sr25519"],"type":["scrypt","xsalsa20-poly1305"],"version":"3"},"meta":{"name":"cere","whenCreated":1611176126091,"isHidden":false}}
+APP_WALLET_PASSPHRASE=Beaufort99$
 AMOUNT=1000000000000000
```

Comment:
Let's apply `SCENARIO_NUMBER` prefix and for `SCENARIO_1` also.

Reply:
changed this to only use one standard AMOUNT. No need to have multiple amounts